VEDA_OS01/models.py

```diff
@@ -229,24 +229,31 @@ def __unicode__(self):
         )
 
 
-class Course(models.Model):
+class Course(TimeStampedModel):
     """
     Model for Course.
```

Comment:
@Qubad786 I meant that we should add the comments about this course i.e. it's not a course run but a whole course.

Reply:
@muzaffaryousaf added.